example/extensions/lib_custom_op/README.md

```diff
@@ -28,26 +28,26 @@ Custom operators (CustomOp) enable users to write new operators without compilin
 
 ### Have MXNet Ready
 
-First you should install MXNet either from compiling from source code or download from nightly build. It doesn’t matter if the build comes with CUDA or MKLDNN. The custom operator doesn’t interact with the execution of other native MXNet operators.
+First you should install MXNet either from compiling from source code or download from nightly build. For running the example below, it doesn’t matter if it is a CUDA, MKLDNN or Vanila build. The custom operator doesn’t interact with the execution of other native MXNet operators. Note that if you want to run GPU examples and write your custom operators running on GPU, you still need MXNet CUDA build.
 
-### Run An Example:
+### Run An Example
 
 You can start getting familiar with custom operators by running some examples provided in the **example/extensions/lib_custom_op** directory. Start with a common linear algebra operator like `gemm` (Generalized Matrix Multiplication). Go to `lib_custom_op` directory and follow these steps:
 
 1. Run `make gemm_lib`. The Makefile will generate a dynamic library **libgemm_lib.so** compiled from `gemm_lib.cc`. This is the library you are going to load that contains everything for the custom gemm operator.
 2. Run `python test_gemm.py`. It’ll first load the above .so library, find the operators, register them in the MXNet backend, print "Found x operators", then invoke the operator like a regular MXNet operator and output the result.
```

Comment:
Change this:
> It’ll first load ...

to:
This program first loads the library compiled in step 1, ...

Reply:
lets add the actual output from running this test so users can compare valid output with what they get rather than just say "Found x operators"